examples/flutter_gallery/lib/demo/shrine/shrine_home.dart

```diff
@@ -39,7 +39,8 @@ class ShrineGridDelegate extends GridDelegate {
     assert(childCount >= 0);
     return new GridSpecification.fromRegularTiles(
       tileWidth: constraints.maxWidth / 2.0 - 8.0,
-      tileHeight: constraints.maxWidth * 0.67,
+      // height = ProductPriceItem + product image + VendorItem
+      tileHeight: 40.0 + 144.0 + 40.0,
```

Comment:
To be fully correct, I'd have to do something like:
ProductPriceItem.padding*2 + defaultFontHeight + 144.0 + VendorItem.padding*2 + defaultFontHeight

I opted for simple hardcoded over introducing a bunch more constants. WDYT?